sensing/autoware_pointcloud_preprocessor/schema/dual_return_outlier_filter_node.schema.json

```diff
@@ -0,0 +1,141 @@
+          "default": "-12.0"
+        },
+        "y_max": {
+          "type": "number",
+          "description": "Maximum of y for `Fixed_xyz_ROI` mode",
+          "default": "2.0"
+        },
+        "y_min": {
+          "type": "number",
+          "description": "Minimum of y for `Fixed_xyz_ROI` mode",
+          "default": "-2.0"
+        },
+        "z_max": {
+          "type": "number",
+          "description": "Maximum of z for `Fixed_xyz_ROI` mode",
+          "default": "10.0"
+        },
+        "z_min": {
+          "type": "number",
+          "description": "Minimum of z for `Fixed_xyz_ROI` mode",
+          "default": "0.0"
+        },
+        "min_azimuth_deg": {
+          "type": "number",
+          "description": "The left limit of azimuth for `Fixed_azimuth_ROI` mode",
+          "default": "135.0",
+          "minimum": 0
+        },
+        "max_azimuth_deg": {
+          "type": "number",
+          "description": "The right limit of azimuth for `Fixed_azimuth_ROI` mode",
+          "default": "225.0",
+          "maximum": 0
+        },
+        "max_distance": {
+          "type": "number",
+          "description": "The limit distance for for `Fixed_azimuth_ROI` mode",
+          "default": "12.0",
+          "minimum": 0
+        },
+        "vertical_bins": {
+          "type": "integer",
+          "description": "The number of vertical bin for visibility histogram",
+          "default": "128",
+          "minimum": 1
+        },
+        "max_azimuth_diff": {
+          "type": "number",
+          "description": "Threshold for ring_outlier_filter",
+          "default": "50",
+          "minimum": 0
+        },
+        "weak_first_distance_ratio": {
+          "type": "number",
+          "description": "Threshold for ring_outlier_filter",
+          "default": "1.004",
+          "minimum": 0
+        },
+        "general_distance_ratio": {
+          "type": "number",
+          "description": "Threshold for ring_outlier_filter",
+          "default": "1.03",
+          "minimum": 0
+        },
+        "weak_first_local_noise_threshold": {
+          "type": "integer",
+          "description": "The parameter for determining whether it is noise",
+          "default": "2",
+          "minimum": 0
+        },
+        "roi_mode": {
+          "type": "string",
+          "description": "roi mode",
+          "default": "Fixed_xyz_ROI"
+        },
+        "visibility_error_threshold": {
+          "type": "number",
+          "description": "When the percentage of white pixels in the binary histogram falls below this parameter the diagnostic status becomes ERR",
+          "default": "0.5",
+          "minimum": 0
+        },
+        "visibility_warn_threshold": {
+          "type": "number",
+          "description": "When the percentage of white pixels in the binary histogram falls below this parameter the diagnostic status becomes WARN",
```

Comment:
This says percentage, yet the value is between 0 and 1

Reply:
Fixed the original description from `percentage` to `proportion` in https://github.com/autowarefoundation/autoware.universe/pull/8475/commits/10e77aa3c7fc4e30054ce57bc62d44dbacf8f362